regtests/client/python/cli/polaris_cli.py

```diff
@@ -106,15 +109,7 @@ def _get_client_builder(options):
             )
         elif has_client_secret:
             builder = lambda: ApiClient(
-                Configuration(host=polaris_management_url, username=options.client_id, password=options.client_secret),
-            )
-        elif os.getenv('CLIENT_ID') and os.getenv('CLIENT_SECRET'):
-            builder = lambda: ApiClient(
-                Configuration(
-                    host=polaris_management_url,
-                    username=os.getenv(CLIENT_ID_ENV),
-                    password=os.getenv(CLIENT_SECRET_ENV)
-                )
+                Configuration(host=polaris_management_url, username=client_id, password=client_secret),
             )
         else:
             raise Exception(f'Please provide credentials via either {Argument.to_flag_name(Arguments.CLIENT_ID)} &'
```

Comment:
Minor suggestion: I'd move this part right under the new line 100 to fail fast, and more readable. 
```
if has_access_token and has_client_secret:
            raise Exception(f'Please provide credentials via either {Argument.to_flag_name(Arguments.CLIENT_ID)} &'
                            f' {Argument.to_flag_name(Arguments.CLIENT_SECRET)} or'
                            f' {Argument.to_flag_name(Arguments.ACCESS_TOKEN)}, but not both')


if not has_access_token and not has_client_secret:
            raise Exception(f'Please provide credentials via ...")
```

Reply:
Good suggestion. Updated.